packages/contracts/hardhat/forks.ts

```diff
@@ -1,12 +1,11 @@
-import { HardhatNetworkForkingUserConfig } from "hardhat/types";
+import { HardhatNetworkForkingUserConfig } from 'hardhat/types';
 
 export type ForkData = HardhatNetworkForkingUserConfig & {
   accounts: Record<string, string>;
 };
 
 const bscRpcUrl = process.env.BSC_MAINNET_RPC_URL;
-if(!bscRpcUrl)
-  throw new Error("Missing BSC_MAINNET_RPC_URL environment variable");
+if (!bscRpcUrl) throw new Error('Missing BSC_MAINNET_RPC_URL environment variable');
```

Comment:
This one lines makes much harder to scan code. Please add rule `"curly": ["error", "multi-or-nest"],`